src/bin/observable.ts

```diff
@@ -199,29 +205,37 @@ try {
     }
   }
 } catch (error: any) {
+  const wrapWidth = process.stdout.columns ?? 100;
```

Comment:
```suggestion
  const wrapWidth = process.stdout.columns ?? 80;
```

We should centralize this, but elsewhere we use 80, and I think that’s more readable than 100.